src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -827,15 +832,13 @@ void readAnalyze(TrackPointer track, double sampleRate, int timingOffset, bool i
     double sampleRateKhz = sampleRate / 1000.0;
     double samples = sampleRateKhz * mixxx::kEngineChannelCount;
 
-    QList<memory_cue_t> memoryCues;
+    QList<memory_cue_loop_t> memoryCuesLoops;
```

Comment:
This variable name is confusing. Is this cues and loops? If so, rename it to `memoryCuesAndLoops`